ios/RCTTabViewViewManager.mm

```diff
@@ -31,5 +31,6 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(labeled, BOOL)
 RCT_EXPORT_VIEW_PROPERTY(ignoresTopSafeArea, BOOL)
 RCT_EXPORT_VIEW_PROPERTY(disablePageAnimations, BOOL)
+RCT_EXPORT_VIEW_PROPERTY(scrollEdgeAppearance, NSString)
```

Comment:
You also need to add it to JS Spec here: https://github.com/okwasniewski/react-native-bottom-tabs/blob/95782ac53c09dd5d42b2ece0aae877310c762857/src/TabViewNativeComponent.ts#L18